types/mongodb/index.d.ts

```diff
@@ -2278,14 +2283,20 @@ export class ChangeStream extends Readable {
     stream(options?: { transform?: Function }): Cursor;
 }
 
+export interface ResumeToken {
+    _data: string;
```

Comment:
This is documented [here](https://docs.mongodb.com/master/changeStreams/#change-stream-resume-token) as being either a `string` or `BinData` (being of type `BSON.Binary`).

More importantly, this is meant to be a relatively opaque type. The intent of [the `ResumeToken` type](https://mongodb.github.io/node-mongodb-native/3.3/api/global.html#ResumeToken) is not meant to document the internals of a resumeToken. Rather it is to provide more clarity than `object` into what should be passed in to `startAfter` and `resumeAfter`.

I'm not sure what the policy is for DefinitelyTyped typings, but I would discourage describing the internals of ResumeToken here.

Reply:
Fair point. I wanted to document that because what you'd usually do is pass the same object you got from the driver back to it. I lack enough experience in TS to know how to express that.

Any ideas?

Reply:
I think an empty class could work

```ts
export class ResumeToken {}
```